### PR TITLE
feat: accept JSON Schema Draft 6 & Draft 7

### PR DESCRIPTION
**Pull request by P0lip**
Needed by https://github.com/stoplightio/platform-internal/pull/6083
Needs https://github.com/stoplightio/json-schema-tree/pull/8
For https://github.com/stoplightio/elements/issues/1116

See my comments to learn more about the changes.

### CODE REVIEW

File: src/__tests__/__snapshots__/index.spec.tsx.snap [hunk 1]
```diff
@@ -1067,15 +1067,15 @@ exports[`HTML Output should match arrays/of-complex-objects.json 1`] = `
                     </div>
                   </div>
                 </div>
+                <div>
```
Comment (P0lip) [inline src/__tests__/__snapshots__/index.spec.tsx.snap:1070]: I hope the order doesn't matter for you.

File: src/components/shared/Validations.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ export const numberValidationNames = [
   'exclusiveMaximum',
 ];
 
-const exampleValidationNames = ['examples', 'example', 'x-example'];
+const exampleValidationNames = ['examples'];
```
Comment (P0lip) [inline src/components/shared/Validations.tsx:28]: `http-spec` will try to produce `examples` at all times now, thus there's no need to handle `example` and `x-example` separately.

File: src/components/shared/Validations.tsx [hunk 3]
```diff
@@ -58,11 +58,8 @@ const createValidationsFormatter = (name: string, options?: { exact?: boolean; n
 };
 
 const validationFormatters: Record<string, (value: unknown) => ValidationFormat | null> = {
-  ['const']: createValidationsFormatter('Allowed'),
```
Comment (P0lip) [inline src/components/shared/Validations.tsx:61]: `json-schema-tree` turns `const` into an enum with a single member

**Review by domagojk — APPROVED** [review]
LGTM

**Pull request merged**